docs/setup.md

```diff
@@ -0,0 +1,47 @@
+## Requirements
+
+This system uses [Actix](https://actix.rs/) web, and Google Cloud APIs (currently vendored).
+
+## Development Guidelines
+Please see the [CONTRIBUTING.md](./CONTRIBUTING.md) docs on commit guidelines and pull request best
+practices.
+
+## Versioning
+The commit hash of the deployed code is considered its version identifier. The commit hash can be retrieved locally via `git rev-parse HEAD`.
+
+## Development Setup
+1. Install Rust. See [rustup.rs](https://rustup.rs/) for how to install on your platform.
+2. Compile Contile using `cargo build`.
+3. Start connection a local ADM instance (run from root of Contile repo):
```

Comment:
```suggestion
3. Start a local ADM instance (run from root of Contile repo):
```

I know we're shifting to "AMP", but the code and environment variables still call it "ADM", so it might be less confusing to keep using "ADM" everywhere for now.